### ISSUE TITLE
How can I use a wav file that is different from the target music as my melody condition, when i fine-tune musicgen-melody?

### ISSUE THREAD

**Issue by Apple-jun** [facebookresearch/audiocraft#434]
Now I know that when I fine-tune musicgen-melody, the form of the dataset is the same as the form of the other text2music models, that is, the target audio and the condition audio to be converted to the chromatogram are the same. So I would like to ask how to make them different? I gave it a try but found the code hard to modify. I would appreciate any help.

**Comment by UtsahaJoshi** [comment]
@Apple-jun I'm currently working on a project where I need to use vocal audio as the input and generate instrumental music as the output using the musicgen-melody model. I understand that typically the target audio and condition audio are the same, but I need them to be different in my case. Any progress regarding this matter?

**Comment by Frei-2** [comment]
@Apple-jun I am trying to fine-tune musicgen following the docs provided in [MUSICGEN.md](https://github.com/facebookresearch/audiocraft/blob/main/docs/MUSICGEN.md#fine-tuning-existing-models), but I encounter a lot of errors and find that the docs is hard to follow. Can I ask how you make the fine-tune command work?

Specifically, when I am trying to run the fine-tune command, I use the `example dataset` provided in `egs/example` and the command I use is:

```bash
dora run solver=musicgen/musicgen_base_32khz model/lm/model_scale=small continue_from=/cache_dir/musicgen-small conditioner=text2music dset=audio/example
```
- I run the command in the root directory of this project, i.e., `/audiocraft-main`.
- `/cache_dir/musicgen-small` is the local path I place pretrained model I download from [facebook/musicgen-small](https://huggingface.co/facebook/musicgen-small) because our server cann't access hugging face.


Thanks in advance for any help with this particular issue and would appreciate any general tips for something else I might be doing wrong.
